src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/configuration/MetadataStoreConfiguration.kt

```diff
@@ -21,7 +21,6 @@ data class MetadataStoreConfiguration(
     var table: String = "NOT_SET",
     var databaseName: String? = "NOT_SET",
     var caCertPath: String? = "NOT_SET",
-    var queryLimit: String? = "NOT_SET",
     var useAwsSecrets: Boolean = true,
     var numberOfParallelUpdates: Int = 10,
     var batchSize: Int = 10_000) {
```

Comment:
shouldn't this become:
```suggestion
    var batchSize: String? = "NOT_SET",) {
```

Reply:
Will do